server/src/universal.ts

```
@@ -342,6 +348,156 @@ function sendGlobalWebSocketMessage(message: { [key: string]: any } | string) {
+        }
+        case "standard": {
+          const room = createSingleplayerRoom(
+            socket,
+            GameMode.StandardSingleplayer
+          );
+          room.addMember(socket);
+          room.startPlay();
+          break;
+        }
+        case "custom": {
+          let validationResult = validateCustomGameSettings(
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'validationResult' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)